flight/mcu/lib/teensyShared/telemetry.cpp

```diff
@@ -16,6 +16,15 @@
 // #define SERIAL_PLOTTING
 
 #include <telemetry.h>
+#include <limits>
+
+
+template<typename T>
+T inv_convert_range(float val, float range){
```

Comment:
Could we have a comment explaining what exactly this function does, because every time I look at it I need to figure it out again? As far as I can tell, it maps a number `val` in the range [0, `range`) into the range [T::min(), T::max()]?